pixart.js

```diff
@@ -1 +1,26 @@
+$(document).ready(function(){
+  console.log("YOOOO")
 
+    var colorInput = $("#color-field").val();
```

Comment:
I would omit `.val()` here. You just want the variable to point to the DOM element, rther than the value inside the DOM element. It's going to define that variable as soon as the page loads, when the value of the input field will be blank or `""`. I think that's why your code may not be working as expected. I think that's all though! 👍